lib/storage/src/rbac/mod.rs

```diff
@@ -70,21 +71,30 @@ impl Access {
     /// explain why the access is granted, e.g. ``Access::full("Internal API")`` or
     /// ``Access::full("Test")``.
     pub const fn full(_reason: &'static str) -> Self {
-        Self::Cluster(ClusterAccessMode::ReadWrite)
+        Self::Global(GlobalAccessMode::Manage)
     }
 
     pub const fn full_ro(_reason: &'static str) -> Self {
-        Self::Cluster(ClusterAccessMode::Read)
+        Self::Global(GlobalAccessMode::Read)
     }
 
-    /// Check if the user has access to the whole cluster.
-    pub fn check_cluster_access(
+    /// Check if the user has global access.
+    pub fn check_global_access(
         &self,
-        min_mode: ClusterAccessMode,
+        min_mode: GlobalAccessMode,
     ) -> Result<CollectionMultipass, StorageError> {
         match self {
-            Access::Cluster(mode) if *mode >= min_mode => Ok(CollectionMultipass),
-            _ => Err(StorageError::unauthorized("Cluster access is not allowed")),
+            Access::Global(GlobalAccessMode::Manage) => Ok(CollectionMultipass),
+            Access::Global(GlobalAccessMode::Read) => {
+                if min_mode == GlobalAccessMode::Read {
+                    Ok(CollectionMultipass)
+                } else {
+                    Err(StorageError::unauthorized(
+                        "Only read-only access is allowed",
+                    ))
+                }
+            }
+            _ => Err(StorageError::unauthorized("Global access is not allowed")),
```

Comment:
Same thing: if the provided access is not `Access::Global` but the user needs global, the message should be the other way around.

```suggestion
            _ => Err(StorageError::unauthorized("JWT needs global access")),
```